WooCommerce/src/main/kotlin/com/woocommerce/android/model/Order.kt

```diff
@@ -98,12 +99,29 @@ data class Order(
         val totalTax: BigDecimal,
         val total: BigDecimal,
         val variationId: Long,
-        val attributesList: String
+        val attributesList: List<Attribute>
     ) : Parcelable {
         @IgnoredOnParcel
         val uniqueId: Long = ProductHelper.productOrVariationId(productId, variationId)
+
         @IgnoredOnParcel
         val isVariation: Boolean = variationId != 0L
+
+        @Parcelize
+        data class Attribute(
+            val key: String,
+            val value: String
+        ) : Parcelable
+
+        /**
+         * @return a comma-separated list of attribute values for display
+         */
+        val attributesDescription
+            get() = attributesList.filter {
```

Comment:
⛏️ Minor nit pick: currently we'll compute this value every time we request it, we can change it to not-backing-field to compute this one. But maybe let's handle it in some other PR to not block this one.

```suggestion
        val attributesDescription = attributesList.filter {
```

Reply:
Usually I try to avoid that too and I like to use by lazy to solve that, but the problem is that using by lazy or direct assignment cause problems with the Parcel structure, so I end up leaving it as an in-line computation instead of a stored field.

Reply:
gotcha! I didn't know that but it make sense - less data for `Parcelize` to handle, the better.